vendor/knative.dev/pkg/apis/test/example/v1alpha1/fiz_types.go

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
just curious why these get pulled into the vendor dir, I thought we pruned?

Reply:
I think the types are checked in, but the codegen isn't.

Reply:
Just seems goofy...

Reply:
yeah that is right, the types are checked. I use them to make sure the output of the generators compiles.